src/main/java/org/atlasapi/media/channel/Channel.java

```
@@ -527,6 +539,11 @@ public Duration getTimeshift() {
     }
 
     @Nullable
+    public Boolean isTimeshifted() {
```

But then can't you have `null` default to `false`? Or is that considered misleading as it might not be accurate?